tests/materialize/materialize-s3-parquet/ingest-data.sh

```diff
@@ -1,16 +1,22 @@
 #!/bin/bash
 set -e
 
-# Sending corresponding data to the websockets.
-websocat --protocol json/v1 "${WEBSOCKET_URL_SIMPLE}" < "${DATASET_SIMPLE}"
+# Pushing data to the capture.
+go run ${DATA_INGEST_SCRIPT_PATH} \
+  --server_address=${CONSUMER_ADDRESS} \
+  --capture=${PUSH_CAPTURE_NAME} \
+  --binding_num=${BINDING_NUM_SIMPLE} \
+  --data_file_path=${DATASET_SIMPLE}
 
-pv --line-mode --quiet --rate-limit 1 < "${DATASET_MULTIPLE_DATATYPES}" \
-      | websocat --protocol json/v1 "${WEBSOCKET_URL_MULTIPLE_DATATYPES}"
+go run ${DATA_INGEST_SCRIPT_PATH} \
+  --server_address=${CONSUMER_ADDRESS} \
+  --capture=${PUSH_CAPTURE_NAME} \
+  --binding_num=${BINDING_NUM_MULTIPLE_DATATYPES} \
+  --data_file_path=${DATASET_MULTIPLE_DATATYPES}
 
 sleep 5
```

Comment:
nit: Add a comment? I don't think it's obvious why this `sleep` is here without one

Reply:
Done.